rusoto/signature/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+authors = [
+    "Anthony DiMarco <ocramida@gmail.com>",
+    "Jimmy Cuadra <jimmy@jimmycuadra.com>",
+    "Matthew Mayer <matthewkmayer@gmail.com>",
+    "Nikita Pekin <contact@nikitapek.in>"
+]
+description = "AWS SDK for Rust - Request Signing"
+documentation = "https://docs.rs/rusoto_signature"
+keywords = ["AWS", "Amazon"]
+license = "MIT"
+name = "rusoto_signature"
+readme = "README.md"
+repository = "https://github.com/rusoto/rusoto"
+version = "0.41.0"
+homepage = "https://www.rusoto.org/"
+edition = "2018"
+
+[badges]
+appveyor = { repository = "matthewkmayer/rusoto", branch = "master" }
+azure-devops = { project = "matthewkmayer/Rusoto", pipeline = "rusoto.rusoto", build="1" }
+
+[build-dependencies]
+rustc_version = "0.2.1"
+
+[dependencies]
+bytes = "0.4.12"
```

Comment:
I copied this from core and trimmed down the dependencies to only what was used in this crate. I imagine in the future this could not even depend on hyper but I think this is fine for now